src/css/controls/imports/icons.less

```diff
@@ -118,3 +118,24 @@
     }
 }
 // stylelint-enable indentation
+.jw-breakpoint--1 {
+    .jw-icon-cc,
+    .jw-icon-cast,
+    .jw-icon-airplay,
+    .jw-icon-settings,
+    .jw-icon-audio-tracks,
+    .jw-icon-hd,
+    .jw-settings-sharing,
+    .jw-icon-fullscreen {
+        display: none;
+    }
+
+    .jw-icon-volume,
```

Comment:
we need to push the icons slightly up to give the main container enough touch space after it gets set to `30px` height. this alleviates the large (but empty) control bar blocking the play button more than it should